examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/PaywallViewActivity.kt

```diff
@@ -14,6 +17,17 @@ import com.revenuecat.purchases.ui.revenuecatui.PaywallListener
 class PaywallViewActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        val windowInsetsController = ViewCompat.getWindowInsetsController(window.decorView)
+        windowInsetsController?.isAppearanceLightNavigationBars = false
+        windowInsetsController?.isAppearanceLightStatusBars = true
+
+        val window = window
+        // For extending content behind the Navigation Bar as well
+        window.addFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS)
+
+        WindowCompat.setDecorFitsSystemWindows(window, false)
```

Comment:
I'm actually wondering if we should make the `PaywallActivity` in the sdk to paint over the status bar... But that can come in a different PR if we decide to do that.